polygeist/tools/cgeist/Test/Verification/sycl/single_task.cpp

```diff
@@ -2,10 +2,6 @@
 
```

Comment:
Do we really need `clang-format off` here? It seems that the `.clang-format` [file](https://github.com/intel/llvm/blob/f26449e6bff15100773151854f143d7b968067df/polygeist/tools/cgeist/Test/.clang-format) in the parent directory already disables clang-format for this directory. 

Also, it seems that the test case is missing the LLVM copyright header, the other tests in the same directory carry the header.

Reply:
In vscode if I change a lit test case (.cpp) my editor reformats it on save (And it fails to read the .clang-format file somehow) so I would like to have the `clang-format-off`.

Reply:
> Also, it seems that the test case is missing the LLVM copyright header, the other tests in the same directory carry the header.

All lit tests in `llvm/polygeist/tools/cgeist/Test/Verification` don't have the LLVM copyright, except the ones in `llvm/polygeist/tools/cgeist/Test/Verification/sycl`.

Reply:
> In vscode if I change a lit test case (.cpp) my editor reformats it on save (And it fails to read the .clang-format file somehow) so I would like to have the `clang-format-off`.

Seems like a different extension for the editor solves @etiotto issue, so maybe we can still remove it from the individual test-file here.

Reply:
> All lit tests in `llvm/polygeist/tools/cgeist/Test/Verification` don't have the LLVM copyright, except the ones in `llvm/polygeist/tools/cgeist/Test/Verification/sycl`. Probably makes more sense to remove the LLVM copyright from the `sycl` directory.

Agreed, I think tests in the LLVM repository tend to not have copyright header, so it should be fine to remove them (optionally in this PR, or at a different time).